src/substrate/types.ts

```diff
@@ -137,10 +137,9 @@ interface IEvent {
  */
 export interface INewSession extends IEvent {
   kind: EventKind.NewSession
-  active: Array<_AccountId>
-  waiting: Array<_AccountId>
-  exposure: Array<Exposure>
-  sessionIndex: SessionIndex
+  activeExposures: {[key: string]: any}
```

Comment:
`any` doesn't mean it's serializable :smirk: you'll need to define a type explicitly here, maybe create a sub-interface or declare an object literal with all the required fields from `Exposure`.